src/data/faq.json

```diff
@@ -610,7 +609,7 @@
                         "textblock": [
                             "Your app regularly checks the certificates stored in the app (including the certificates stored there for other people) in order to be able to pass on individually compiled recommendations to you (or the other people concerned). This check is performed directly in your app, so that no data leaves the smartphone in the process. The rules used and the possible recommendation texts are downloaded from the rule server beforehand together with the current updates of the app, so that no individual information about you or your certificates (or about other persons and their certificates) is revealed.",
                             "If the rules established in accordance with the respective current recommendations of the Standing Committee on Vaccination at the Robert Koch Institute (STIKO) apply to your certificates stored in the app, the app will notify you of the corresponding vaccination recommendation. Please then discuss with your doctor(s) whether you should receive another vaccine dose of a COVID-19 vaccine.",
-                            "If the assumptions derived from your saved certificates do not apply because you have already received further COVID-19 vaccinations or have undergone a proven SARS-CoV-2 infection, you can ignore this notification. In this case, we recommend scanning the relevant vaccination or recovery certificates into the app.",
+                            "If the assumptions derived from your saved certificates do not apply because you have already received further COVID-19 vaccinations or have undergone a proven SARS-CoV-2 infection, you can ignore this notification. In this case, we recommend scanning the relevant vaccination or certificates of recovery into the app.",

```

Comment:
This change does not work grammatically. I suggest reverting to the original text.

Reply:
Fixed with [`a128219`](https://github.com/corona-warn-app/cwa-website/pull/2491/commits/a1282197b8ade5f9592a569015922557e715ceaf).